src/plugins/visualize/public/application/utils/index.ts

```diff
@@ -17,12 +17,8 @@
  * under the License.
  */
 
-import { i18n } from '@kbn/i18n';
-
-export const defaultFeedbackMessage = i18n.translate('kibana_utils.defaultFeedbackMessage', {
```

Comment:
Instead of exporting `defaultFeedbackMessage` from `kibana_utils` to every experimental vis (`input_control`, `vega`, `metrics`), I moved it into visualize: https://github.com/elastic/kibana/pull/67848/files#diff-1d6c318a6f1b7dcab619931a74884297